src/FakeItEasy/Creation/DummyValueResolver.cs

```diff
@@ -188,32 +193,25 @@ private class ResolveByCreatingLazyStrategy : ResolveStrategy
                 if (typeOfDummy.GetTypeInfo().IsGenericType && typeOfDummy.GetGenericTypeDefinition() == typeof(Lazy<>))
                 {
                     var typeOfLazyResult = typeOfDummy.GetGenericArguments()[0];
-                    var creationResult = resolver.TryResolveDummyValue(typeOfLazyResult, resolutionContext);
-                    object lazyResult = creationResult.WasSuccessful
-                        ? creationResult.Result
-                        : typeOfLazyResult.GetDefaultValue();
-
                     var funcType = typeof(Func<>).MakeGenericType(typeOfLazyResult);

```

Comment:
In the diff view, it looks like funcType is instantiated but no longer used - is that right?

Reply:
I think you're correct, @TimLovellSmith.

Reply:
Indeed. Thanks @TimLovellSmith!